src/sentry/eventstream/kafka/protocol.py

```diff
@@ -20,6 +20,9 @@ def basic_protocol_handler(unsupported_operations):
     # so this function builds a handler function that can deal with both.
 
     def get_task_kwargs_for_insert(operation, event_data, task_state=None):
+        if task_state and task_state.get('skip_consume', False):
+            return None  # nothing to do
```

Comment:
Didn't version bump the protocol since this is purely an additive thing with an obvious default. Open to different views here, though.